pkg/amphora/client.go

```diff
@@ -55,12 +79,47 @@ type Client struct {
 	HTTPClient http.Client
 }
 
-const secretShareURI = "/intra-vcp/secret-shares"
+const secretShareURI = "/secret-shares"
+
+//GetObjectList query secretIds based on the filter argument (tags)
+func (c *Client) GetObjectList(objectListRequestParams *ObjectListRequestParams) (MetadataPage, error) {
```

Comment:
I guess this method is named GetObjectList according to the respective method name in the `Amphora` service. Seems like this method made it through refactoring somehow and should actually be called sth, like `GetSecretList`. 
(_secrets_ were called _objects_ in the beginning..)